contrib/docker/scripts/dbInit.js

```diff
@@ -0,0 +1,19 @@
+const config = {
```

Comment:
Is this file still being used now that we're creating the cluster in healthcheck?